public/google-maps.js

```diff
@@ -81,5 +81,14 @@ function initAutocomplete() {
         const sentimentChart = new Chart($chart, newChart)
       })
 
+    fetch('/database')
```

Comment:
`/database` isn't very descriptive. What's being fetched? That should be the URL path.